server/xpub/entities/manuscript/resolvers.js

```diff
@@ -105,19 +105,29 @@ const resolvers = {
       manuscript.status = 'QA'
       await Manuscript.save(manuscript)
 
-      const user = await User.find(ctx.user)
-      if (user.email !== modifiedManuscriptInput.author.email) {
-        mailer
-          .send({
-            to: modifiedManuscriptInput.author.email,
-            text: 'Please verify that you are a corresponding author',
+      mecaExport(manuscript.id, ctx.user, ctx.ip)
+        .then(() =>
+          logger.info(`Manuscript ${manuscript.id} successfully exported`),
```

Comment:
@hdrury1 Do we definitely not want to send an email to the author, confirming that their submission has been sent?

Normally I would agree that this is redundant, since they can just look at their dashboard. But if it's the case that some authors aren't necessarily "digital natives" or if their primary workflow is geared around using their emails, perhaps they would prefer to use their emails to track their submissions rather than being required to log in to our app?

Just a thought. What do you reckon?

Reply:
It's been considered at length, but ultimately decided against for now. Obviously this is subject to change post-first release depending on real user feedback.